Задание_10/Задание_10/Program.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+
+
+namespace Задание_10
+{
+    class Program
+    {
+        static void conclusion(int[,] rth, int rows, int columns)
+        {
+
+            for (int i = 0; i < rows; i++)
+            {
+                for (int j = 0; j < columns; j++)
+                {
+                    Console.Write(rth[i, j] + "\t");
+                }
+                Console.WriteLine();
+            }
+        }
+        static void Main()
+        {
+            int m, n;
+            Console.WriteLine("Введите кол. столбцов");
+            n = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Введите кол. строчек");
+            m = Convert.ToInt32(Console.ReadLine());
+            int[,] mas = new int[m, n];
+            Random nomber = new Random();
+            for (int i = 0; i < m; i++)
+            {
+                for (int j = 0; j < n; j++)
+                {
+                    mas[i, j] = nomber.Next(0, 99);
+
+                }
+            }
+
+            conclusion(mas, m, n);
+            Console.WriteLine("Введите число");
+            int x;
+            x = Convert.ToInt32(Console.ReadLine());
```

Comment:
Объявление переменной и присваивание ей значения можно делать в одном операторе. Также лучше использовать метод `int.Parse()` для интерпретации строки как целого числа
```suggestion
            int x = int.Parse(Console.ReadLine());
```